src/components/_General/_FormikElements/Checkbox.tsx

```diff
@@ -0,0 +1,83 @@
+import React from 'react';
+
+import styled from '@emotion/styled';
+import { FieldHookConfig, useField } from 'formik';
+import { V } from 'util/theming';
+
+const Container = styled.label`
+  display: flex;
+  align-items: center;
+
+  position: relative;
+  cursor: pointer;
+  font-size: ${V.font.p};
+  color: ${V.color.frontSofter};
+  -webkit-user-select: none;
```

Comment:
We should probably add `PostCSS` to our webpack/babel config so that we don't need to use browser prefixes.